sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/CosmosTemplate.java

```diff
@@ -57,6 +57,7 @@
 public class CosmosTemplate implements CosmosOperations, ApplicationContextAware {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CosmosTemplate.class);
+    private static final String ETAG_KEY = "_etag";
```

Comment:
this constant is defined in a number of places - what about just making the one on CosmosEntityInformation public?

Reply:
fixed. moved to Constants class and used it in all the relevant places